app/assets/stylesheets/application.css

```
@@ -2116,9 +2116,13 @@ img.filecontent.image {background-image: url(/transparent.png);}
 .reaction-button.reacted:hover .icon-svg {
   fill: #c61a1a;
 }
+.reaction-button.reacted:hover, .reaction-button.reacted:active {
```

```suggestion
.reaction-button:hover, .reaction-button:active {
```

`.reacted` は、「自分がリアクションしている」状態を指します。そのため、これがあると、自分がリアクションしていないときは下線が表示去る状態になっています。